cognite_toolkit/_cdf_tk/client/api/project.py

```diff
@@ -0,0 +1,27 @@
+from typing import TYPE_CHECKING
+
+from cognite.client import CogniteClient
+
+from cognite_toolkit._cdf_tk.client.data_classes.project import ProjectStatusList
+from cognite_toolkit._cdf_tk.utils.http_client import HTTPClient, ParamRequest
+
+if TYPE_CHECKING:
+    from cognite_toolkit._cdf_tk.client import ToolkitClientConfig
+
+
+class ProjectAPI:
+    def __init__(self, config: "ToolkitClientConfig", cognite_client: CogniteClient) -> None:
+        self._config = config
+        self._cognite_client = cognite_client
+        self._http_client = HTTPClient(config, split_items_status_codes=set())
+
+    def status(self) -> ProjectStatusList:
+        """Retrieve information about the current project."""
```

Comment:
Nit (non-blocking): This is inaccurate. It now returns list of states of the projects the user has access to, and then the user calls this_project to find the particular one to get the correct project. It is tempting to do the filtering automatically already here, but then there wouldn't be a way to access all the other projects you have access to, which can be useful, so I would instead change the docstring. That can be done in a follow-up, though.

Reply:
It is good point, will do it in a follow up, as this is internal and not visible to the und-user.